sklearn/metrics/cluster/supervised.py

```diff
@@ -415,17 +415,17 @@ def completeness_score(labels_true, labels_pred):
     Non-perfect labelings that assign all classes members to the same clusters
     are still complete::
 
-      >>> print(completeness_score([0, 0, 1, 1], [0, 0, 0, 0]))
-      1.0
-      >>> print(completeness_score([0, 1, 2, 3], [0, 0, 1, 1]))
+      >>> completeness_score([0, 0, 1, 1], [0, 0, 0, 0])
       1.0
+      >>> completeness_score([0, 1, 2, 3], [0, 0, 1, 1]) # doctest: +ELLIPSIS
+      0.99999999...
```

Comment:
Do we have a better way here? 0.99999999... seems a bit magical to serve as an example for ordinary users.

Reply:
Is this a bug or a feature?

Reply:
> Do we have a better way here? 0.99999999... seems a bit magical to serve as an example for ordinary users.

Not that I know off.

> Is this a bug or a feature?

It is kind of a feature. Now numpy floats prints and reprs the same as Python floats.